### PR TITLE
Calendar tests and styling

### PR DESCRIPTION
**Pull request by Artorp**
As part of https://github.com/IT2810/it2810-webutvikling-h18-prosjekt-3-27/issues/4

- Adds snapshot and unit tests for the Calendar feature
- Updates style of Calendar subscreens
- Updates test script to run jest through node directly

### CODE REVIEW

**Review by hellevandenbroek — DISMISSED** [review]
Looks very good!

File: components/Calendar/AddAgendaScreen.js [hunk 1]
```diff
@@ -73,7 +90,14 @@ const styles = StyleSheet.create({
     alignItems: "center",
     justifyContent: "flex-start"
```
Comment (SanderBreivik) [inline components/Calendar/AddAgendaScreen.js]: Maybe add background color "white"

File: components/Calendar/AddAgendaScreen.js [hunk 2]
```diff
@@ -42,24 +43,40 @@ class AddAgendaScreen extends Component {
...
-          style={styles.textInput}
+          style={[styles.margins, styles.textInput]}
           value={this.state.note}
-          defaultValue={"A short description..."}
+          multiline={true}
+          numberOfLines={5}
+          underlineColorAndroid="transparent"
+          textAlignVertical="top"
+          placeholder={"Enter description..."}
           onChangeText={this.handleNoteChange}
```
Comment (SanderBreivik) [inline components/Calendar/AddAgendaScreen.js:78]: autoComplete=false to remove annoying autocomplete

File: components/Calendar/AddAgendaScreen.js [hunk 3]
```diff
@@ -42,24 +43,40 @@ class AddAgendaScreen extends Component {
...
+        </Text>
+        <Text style={styles.margins}>Name:</Text>
         <TextInput
-          style={styles.textInput}
+          style={[styles.margins, styles.textInput]}
           value={this.state.name}
-          defaultValue={"Name"}
+          underlineColorAndroid="transparent"
+          placeholder={"Name"}
           onChangeText={this.handleNameChange}
```
Comment (SanderBreivik) [inline components/Calendar/AddAgendaScreen.js:66]: autoComplete=false, to remove annoying auto completion

File: components/Calendar/AgendaPersistence.test.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  test("deleteAgenda should delete agenda", async (done) => {
+    const timeStringDate = "2018-09-16";
+    const agenda = {
+      date: new Date(timeStringDate),
+      name: "My Agenda",
+      note: "A longer description of the agenda",
+      id: 25
+    };
+    await AgendaPersistence.saveAgenda(agenda);
+    await AgendaPersistence.deleteAgenda(agenda);
```
Comment (SanderBreivik) [inline components/Calendar/AgendaPersistence.test.js:58]: Maybe check that the agenda is actually saved before deletion.

**Review by SanderBreivik — CHANGES_REQUESTED** [review]

**Review by SanderBreivik — APPROVED** [review]

**Pull request merged**